src/functions/vacation-request/update-vacation-request/handler.ts

```diff
@@ -93,6 +97,15 @@ const updateVacationRequestHandler: ValidatedEventAPIGatewayProxyEvent<
         updatedStatus: currentStatus
       });
     }
+    if (draftStatusChanged) {
```

Comment:
Could a user have a vacation that has been submitted and then change it to a draft? In which case this would send a draft to an admin right?

Probably we should ensure that it is not possible to change a submitted vacation to draft. Please check, if not lets create a card in the backlog for this.